spec/rubocop/cli_spec.rb

```diff
@@ -1627,6 +1627,23 @@ def abs(path)
       end
     end
 
+    describe '-e/--extra-details' do
+      it 'shows extra details' do
```

Comment:
right - are there some examples of those kinds of tests? that was what I wanted to do - actually test a change in configuration ...